doc/reference/autoinstall-reference.rst

```diff
@@ -635,10 +635,9 @@ password
 The password for the new user, encrypted. This is required for use with
 ``sudo``, even if SSH access is configured.
 
-The crypted password string must conform to what
-`passwd <https://manpages.ubuntu.com/manpages/passwd.1.html>`_
-expects. Depending on the special characters in the password hash, quoting may
-be required, so it's safest to just always include the quotes around the hash.
+The crypted password string must conform to what the
+``passwd`` command requires. See the `passwd(1) manual page <https://manpages.ubuntu.com/manpages/lunar/en/man1/passwd.1ssl.html>`_
```

Comment:
This is more of a problem for this link because the redirect from https://manpages.ubuntu.com/manpages/passwd.1.html is currently wrong (I read a discussion about it IIRC). What it should point to is the passwd(1) manpage - which can be found here for instance: https://manpages.ubuntu.com/manpages/lunar/en/man1/passwd.1.html

If we're hardcoding the links to lunar, https://manpages.ubuntu.com/manpages/lunar/en/man1/passwd.1.html is the one we should use instead.

Reply:
On second thought, maybe we want passwd(5) instead, I'm not sure.

Reply:
> what it should point to is the passwd(1) manpage

Ugh, yes, I saw that but then forgot about it. Fixing it now.